.github/workflows/test_sllm_store.yaml

```diff
@@ -0,0 +1,51 @@
+name: Test Sllm Store
+
+on: workflow_dispatch
```

Comment:
Seems that the [`workflow_dispatch`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#workflow_dispatch) means it only works when manually triggered? Shall we change it to [`pull_request`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) so that it could be automatically tiggered and will be one of the conditions in PR review?

Reply:
We plan to enable GPU tests on self-hosted GPU runners. However, running these tests automatically can be too costly. Therefore, I propose that all GPU tests be triggered manually, running only once when everything else is ready.

Reply:
If there are better ways to reduce GPU cost, pls lmk.

Reply:
Sure, for GPU tests, manually trigger may be fine. But I suppose that this PR is CPU only? (as mentioned in the title of the PR)

Reply:
OK, I'll delete GPU tests from this pr, and revert it back to auto-trigger.